airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -477,6 +481,32 @@ public int getNumberOfFileBuffers(final JsonNode config) {
   }
 
   public static void main(final String[] args) throws Exception {
+    if ("--write".equals(args[0])) {
+      final List<String> targetStrings = List.of("foo", "bar", "baz");
+      final BigQueryException e = new BigQueryException(42, "Not found: Table bar.baz was not found in location US at [53:5]");
+
+      final String mangledMessage =
+          "airbyte_destination_bigquery_error: "
+          + targetStrings.stream().reduce(
+              e.getMessage().replaceAll("\\d", "?"),
+              (message, targetString) -> message.replace(targetString, "?"));
+
+      // This puts the mangled string into the trace external message and the original string into the internal message, which is backwards
+      // we'll need to add a new utility method to this class that does the right thing
+//      AirbyteTraceMessageUtility.emitSystemErrorTrace(e, mangledMessage);
+      final Consumer<AirbyteMessage> outputRecordCollector = Destination::defaultOutputRecordCollector;
+      outputRecordCollector.accept(new AirbyteMessage()
+          .withType(AirbyteMessage.Type.TRACE)
+          .withTrace(new AirbyteTraceMessage()
+              .withEmittedAt((double) System.currentTimeMillis())
+              .withType(AirbyteTraceMessage.Type.ERROR)
+              .withError(new AirbyteErrorTraceMessage()
+                  .withMessage(e.getMessage())
+                  .withInternalMessage(mangledMessage)
```

Comment:
these two lines are the interesting piece - we display the original exception message in the UI, but pass the mangled message into the SentryEvent.message field